apps/earn-protocol/helpers/vault-decorators.ts

```
@@ -289,5 +312,29 @@ export const decorateWithHistoricalChartsData = (
+    ...vault,
+    customFields: {
+      ...vault.customFields,
+      arksInterestRates: Object.keys(arkInterestRatesMap).reduce<{
+        [key: string]: number
+      }>((acc, key) => {
+        const interestRates = arkInterestRatesMap[key]
+
+        console.log(key, interestRates.latestInterestRate[0].rate[0].rate)
+
+        return {
+          ...acc,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)